### ISSUE TITLE
black theme. compare files.

### ISSUE THREAD

**Issue by A-Kolotsey** [doublecmd/doublecmd#462]
when compare files. unreadable compare text color on black theme
![dc-compare](https://user-images.githubusercontent.com/72033647/173061598-d4c7b07c-bfc3-46e9-a06a-f7630b33209a.jpg)
![dc-compare2](https://user-images.githubusercontent.com/72033647/173062005-1fdaab0b-8cc1-4f78-9b8a-80d8c4eb72d2.jpg)
.

**Comment by Skif-off** [comment]
@A-Kolotsey,
currently DC does not switch colors automatically (and I do not know if it will do so in the future), but you can try to change the colors [manually](http://doublecmd.github.io/doc/en/configxml.html), see `<Added>`, `<Deleted>` and `<Modified>`. (The color format is described before the table.) Or try to use [this values](https://github.com/j2969719/doublecmd-plugins/tree/master/dark-theme).


@FilthyChef,
your problem has nothing to do with the dark mode: fonts for the internal editor and viewer must be monospace.

**Comment by alexx2000** [comment]
I plan to add two sets of colors which will auto switched by window background color. Related issue #440.

**Comment by oomek** [comment]
It's somehow related so I haven't opened a separate issue.
I've managed to tune the colors for Differ in doublecmd.xml, but how can I do the same for SyncDirs?
Pure RGB colors on a dark backgorund look awful.

![image](https://user-images.githubusercontent.com/2974860/221194235-63180275-bca1-493c-9aec-23b0ba65c9e3.png)

**Comment by alexx2000** [comment]
See `<SyncDirs><Colors>` [here](https://doublecmd.github.io/doc/en/configxml.html#manualed).

**Comment by oomek** [comment]
Aah I see. I thought <Left> <Right> are colors for left and right pane in general, not for differencies.
BTW, Would be really nice if the colors were stored as hex not integers and reversed.

**Comment by alexx2000** [comment]
Fixed in the latest 1.1.0.

**Comment by pkzc** [comment]
Differ colors  of 1.1.9 still do not work correctly in dark mode: the added and deleted lines are colored  all right by the colors set up for the dark mode, but the changed lines use the light mode color scheme when "Line differences' is turned on.

First I tried to set the Differ colors for Light in colors.js equal to those for Dark, but it didn't help.

Then I removed the same light scheme differ colors from doublecmd.xml, and it didn't help either: DC seems to have a copy of those light mode differ colors hard coded somewhere.

When I switch off and on 'Paint Background' then the changed lines get colored correctly, but this is a very inconvenient workaround.
